test/test_visualizer.py

```diff
@@ -0,0 +1,249 @@
+# Copyright 2022 Kenji Brameld
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import rclpy
+from rclpy import Parameter
+
+from soccer_vision_3d_msgs.msg import (
+    Ball, BallArray, FieldBoundary, GoalpostArray, MarkingArray, MarkingSegment, ObstacleArray,
+    Robot, RobotArray)
+from soccer_vision_3d_rviz_markers.visualizer import SoccerVision3DMarkers
+from visualization_msgs.msg import Marker, MarkerArray
+
+
+class TestVisualizer:
+    """Tests against the SoccerVision3DMarkers class."""
+
+    received = None
+
+    def _callback_msg(self, msg):
+        self.received = msg
+
+    def test_parameters_default_values(self):
+        rclpy.init()
+        visualizer_node = SoccerVision3DMarkers()
+        assert visualizer_node.get_parameter('ball_diameter').value == 0.10
+        assert visualizer_node.get_parameter('marking_segment_width').value == 0.05
+        rclpy.shutdown()
+
+    def test_topics(self):
+
+        rclpy.init()
```

Comment:
Why are there sometimes newlines in methods? Applies to this file and `test_conversion.py`.

Reply:
Are you referring to the empty line between ``def test_topics(self):``and ``rclpy.init()``?
I don't have a strong preference, but I believe it increases readability for me, and doesn't go against any python guidelines. Is it unnatural to you...?

Reply:
It's just something I never really cared about, but now that you've pointed out ..... the inconsistency is starting to bother me :sweat_smile: